data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/teku/PeerScore.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.response.v1.teku;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.Objects;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class PeerScore {
+
+  @JsonProperty("peer_id")
+  @Schema(
+      type = "string",
+      description =
+          "Cryptographic hash of a peer’s public key. "
+              + "'[Read more](https://docs.libp2p.io/concepts/peer-id/)",
+      example = "QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N")
+  public final String peerId;
+
+  @JsonProperty("gossip_score")
+  @Schema(
+      type = "string",
+      format = "number",
+      description = "Gossip score for the associated peer.",
+      example = "1.0")
```

Comment:
(nit) I'd add a decimal value to - this is currently displaying as "1" in the generated swagger docs (http://localhost:5051/swagger-ui):
```suggestion
      example = "1.2")
```

Reply:
I understand, thank you! In order to resolve conflicts, can I just merge branch ConsenSys:master into BogdanStirbat:make-gossip-scores-visible, then push the changes?

Reply:
I improved the documentation. Thank you!

Reply:
> In order to resolve conflicts, can I just merge branch ConsenSys:master into BogdanStirbat:make-gossip-scores-visible, then push the changes?

Yep!  If you have any issues, let me know and I can have a look at the conflicts.

Reply:
I understand. I did the merge. Thank you for your help!